invoicer_project/static/js/invoices_list_request.js

```diff
@@ -0,0 +1,124 @@
+const host = "http://127.0.0.1:8000";
+import {obtainUserInitials, obtainNewAccessToken, addCheckboxesListener, getUserData, addDeleteButtonListeners} from './request_utils.js';
+import {search} from './request_utils.js';
+document.getElementById("search_bar").addEventListener('keyup', ()=> { search('invoice_name', 'client_list_item')});
+
+function createInvoiceListContent(data) {
+    for (let i = 0; i < data.length; i++) {
+        let invoiceName = data[i]["name"];
```

Comment:
Is it possible to use here [destructing assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)?
For example:
```js
const {id, name, price, date_of_payment} = data[i]
```
or
```js
const {id: invoiceId, name: invoiceName, price: invoicePirce, date_of_payment: invoiceDate} = data[i]
```

Reply:
Done